README.md

```diff
@@ -1,7 +1,115 @@
-# Mattermost Boards (Focalboard Plugin)
+# Mattermost Boards
 
-**[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
+### A self-hosted plugin for managing boards withing a Mattermost installation
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+Mattermost boards plugins is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
-To build your own version of Matterboard Boards and upload it to your own Mattermost server, follow the instructions [here](https://developers.mattermost.com/contribute/focalboard/mattermost-boards-setup-guide/).
+## Try Mattermost Boards Plugin 
+
+Access the latest releases of the mattermost boards plugin by downloading the `mattermost-plugin-focalboard.tar.gz` file from the releases in this repository: <https://github.com/mattermost/mattermost-plugin-boards/releases>. After downloading and installing the plugin in the System Console, select the menu in the top left corner and select **Boards**. 
```

Comment:
@marianunez - Does the plugin binary have "focalboard" in the name intentionally? Is there value in renaming to "boards"?

Reply:
We can rename the new bundle file but we can't rename the plugin id because of upgrades. @Rajat-Dabade please confirm the bundle name 
```suggestion
Access the latest releases of the mattermost boards plugin by downloading the `mattermost-plugin-boards.tar.gz` file from the releases in this repository: <https://github.com/mattermost/mattermost-plugin-boards/releases>. After downloading and installing the plugin in the System Console, select the menu in the top left corner and select **Boards**. 
```

Reply:
@marianunez Sure, it's just a part of the config file, changing the name from `focalboard` to `boards` can fix it. Will do it in this PR.

Reply:
@Rajat-Dabade - Would changing ``id`` impact backwards compatibility in any way?

Reply:
@cwarnermm Yup, it will impact, if the customer has an older version plugin, then it won't be upgraded to the newer version because the comparison is done using `Id`.

Reply:
Sure. @cwarnermm, let me try it on my local and I will discuss this with @marianunez once she is back.